sdk/maintenance/Microsoft.Azure.Management.Maintenance/tests/Properties/AssemblyInfo.cs

```diff
@@ -20,5 +20,5 @@
 
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("6D17E66C-92BB-455C-A150-45922A0BA619")]
-[assembly: AssemblyVersion("1.1.0.0")]
-[assembly: AssemblyFileVersion("1.1.0.0")]
+[assembly: AssemblyVersion("1.2.0.0")]
```

Comment:
You should update the `AssemblyInfo.cs` in your `src/` not `tests/`. Test libraries are never published so their assemblyinfo need not be updated

Reply:
Seems we were missing the assemblyinfo, thanks for catching